apps/polarion/polarion_utils.py

```diff
@@ -19,31 +25,74 @@ def git_diff_lines():
         LOGGER.debug(line)
         if line.startswith("+"):
             diff.setdefault("added", []).append(line)
-
+        if line.startswith("-"):
+            diff.setdefault("removed", []).append(line)
     return diff
 
 
 def validate_polarion_requirements(
-    polarion_test_ids,
     polarion_project_id,
+    polarion_test_ids=None,
```

Comment:
```suggestion
    polarion_test_ids=None,
```
Why this is None?
The function do nothing if this is None